content/post/go-git-v4.md

```diff
@@ -0,0 +1,73 @@
+---
+author: campoy
+date: 2017-12-07
+title: "Announcing the latest go-git!"
+image: /post/go-git-v4/go-git.png
```

Comment:
This is how the banner image looks like.
![image](https://user-images.githubusercontent.com/2437584/33715418-b80b35ea-db52-11e7-9a21-effe5fc32214.png)

I'm not sure if @ricardobaeta would like to review it.